src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -664,27 +672,31 @@
             default { "" }
             disablewhen "{ include_adaptor_wheels == 0 }"
             parmtag { "script_callback_language" "python" }
+            help    "The directory for adaptor wheels"
         }
         parm {
              name    "auto_unlock_rops"
              label   "Automatically unlock ROPs"
              type    toggle
              default { "0" }
              parmtag { "script_callback_language" "python" }
+             help    "A locked ROP will use existing outputs and won't re-render. It will also block dependencies from re-rendering. Select this option to automatically unlock dependency ROPs"
          }
          parm {
              name    "auto_parse_hip"
              label   "Automatically parse Hip references"
              type    toggle
              default { "0" }
              parmtag { "script_callback_language" "python" }
+             help    "Automatically parse the input and output file names and directories based on the ROP graph during job submission"
```

Comment:
```suggestion
             help    "Automatically parse the input and output files and directories based on the ROP graph on job submission"
```

I believe this explicitly runs the "parse files" when you press submit, so this feels a bit better